### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by Nyrest**
Link to issue(s) this covers

### Problem
Chinese translation is outdated

### Solution
* Add Chinese translation for "Theme" #2906 
* Translate new decompiler options
* Fix trailing space and punctuation to match the original format.
image

### CODE REVIEW

**Comment by christophwille** [comment]
@maikebing Got time for a quick look at this?

File: ILSpy/Properties/Resources.zh-Hans.resx [hunk 1]
```diff
@@ -417,6 +429,9 @@
   <data name="DecompilerSettings.RemoveOptionalArgumentsIfPossible" xml:space="preserve">
     <value>如果可能，删除可选参数</value>
   </data>
+  <data name="DecompilerSettings.RequiredMembers" xml:space="preserve">
+    <value>Required 必须的成员</value>
```
[the remark this answers is not in the document]
Comment (Nyrest) [reply ILSpy/Properties/Resources.zh-Hans.resx]: 听起来的确更好！

**Review by maikebing — APPROVED** [review]

**Comment by maikebing** [comment]
LGTM

**Comment by christophwille** [comment]
Thanks!

**Pull request merged**